bin/build_container_image

```diff
@@ -2,7 +2,8 @@
 
 REGISTRY=${REGISTRY:-"docker.io"}
 ORGANIZATION=${ORGANIZATION:-"manageiq"}
-TAG=${TAG:-"latest"}
+[[ $TRAVIS_BRANCH == "master" ]] && TAG=latest
```

Comment:
We still want to use the TAG if it was passed in, right? If so, then I think this should be

```suggestion
[[ $TRAVIS_BRANCH == "master" ]] && TAG=${TAG:-latest}
```

Or perhaps it's better to be more explicit?  Something like:

```sh
DEFAULT_TAG="latest"
[[ $TRAVIS_BRANCH != "master" ]] && DEFAULT_TAG="$DEFAULT_TAG-$TRAVIS_BRANCH"
TAG=${TAG:-"$DEFAULT_TAG"}
```